src/extension.ts

```diff
@@ -534,7 +534,19 @@ export class Extension implements RunHooks {
     };
 
     if (mode === 'debug') {
-      await model.debugTests(items, testListener, testRun.token);
+      const debugEnd = new this._vscode.CancellationTokenSource();
```

Comment:
I now think we should move this code to the place that calls `vscode.debug.startDebugging`, instead of having it here.